### ISSUE TITLE
[Accessibility] Status bar footer needs aria-label info

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227257]
-     VS Code Version: 1.88.0
-     OS Version: MacOS Sonoma (v14.6.1)

Issue: The status bar <footer> element in VSCode provided with role="status" and tabindex="0" lacks name information. 

Impact: Screen reader users will have difficulty determining the purpose and state of these controls.

Recommendation: Provide descriptive name as "Status bar" (aria-label = "Status bar") or similar for focusable element with role="status".

[Compliant Code Example]
```
<footer aria-label="Status bar" class="part statusbar status-border-top" id="workbench.parts.statusbar" role="status" aria-live="off" data-keybinding-context="8" tabindex="0" (...)>
(...)
</footer>
```
Screenshot of issue:
<img width="1784" alt="Screenshot 2024-09-02 at 10 00 56 PM" src="https://github.com/user-attachments/assets/426a9121-b6b0-471e-b80b-687ead55dada">

**Comment by meganrogge** [comment]
the bug reproduces when focus is placed on the status bar via the command "focus status bar" then shift tab

**Comment by meganrogge** [comment]
Closing as I cannot reproduce, and no users have reported this.
